### PR TITLE
Add searchbar options in all viewConfigs

### LINKED ISSUE #178: Show gene/genomic interval search bar
Make sure it is in all the view configs

### PR DESCRIPTION
**Pull request by sehilyi**
Fixes #178

### CODE REVIEW

File: src/viewconfigs/horizontal-multivec-1.json [hunk 1]
```diff
@@ -15,6 +15,13 @@
           2671196920.8697634,
           2671320441.9350777
         ],
+        "genomePositionSearchBoxVisible": true,
+        "genomePositionSearchBox": {
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```
Comment (keller-mark) [inline src/viewconfigs/horizontal-multivec-1.json]:
```suggestion
          "chromInfoId": "hg38"
```
I think we want the hg38 assembly for all of these

File: src/viewconfigs/horizontal-multivec-2.json [hunk 2]
```diff
@@ -15,6 +15,13 @@
           2002993569.3469784,
           2011171047.0976794
         ],
+        "genomePositionSearchBoxVisible": true,
+        "genomePositionSearchBox": {
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```
Comment (keller-mark) [inline src/viewconfigs/horizontal-multivec-2.json]:
```suggestion
          "chromInfoId": "hg38"
```

File: src/viewconfigs/horizontal-multivec-3.json [hunk 3]
```diff
@@ -15,12 +15,12 @@
...
         "genomePositionSearchBox": {
-          "autocompleteServer": "//higlass.io/api/v1",
-          "autocompleteId": "OHJakQICQD6gTD7skx4EWA",
-          "chromInfoServer": "//higlass.io/api/v1",
-          "chromInfoId": "hg19",
-          "visible": true
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```
Comment (keller-mark) [inline src/viewconfigs/horizontal-multivec-3.json]:
```suggestion
          "chromInfoId": "hg38"
```

File: src/viewconfigs/horizontal-multivec-4.json [hunk 4]
```diff
@@ -15,6 +15,13 @@
           2001105169.029666,
           2013059447.4149919
         ],
+        "genomePositionSearchBoxVisible": true,
+        "genomePositionSearchBox": {
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```
Comment (keller-mark) [inline src/viewconfigs/horizontal-multivec-4.json]:
```suggestion
          "chromInfoId": "hg38"
```

File: src/viewconfigs/horizontal-multivec-5.json [hunk 5]
```diff
@@ -15,6 +15,13 @@
           2002993569.3469784,
           2011171047.0976794
         ],
+        "genomePositionSearchBoxVisible": true,
+        "genomePositionSearchBox": {
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```
Comment (keller-mark) [inline src/viewconfigs/horizontal-multivec-5.json]:
```suggestion
          "chromInfoId": "hg38"
```

File: src/viewconfigs/horizontal-multivec-6.json [hunk 6]
```diff
@@ -15,6 +15,13 @@
           2671171560.5435214,
           2671345802.2613196
         ],
+        "genomePositionSearchBoxVisible": true,
+        "genomePositionSearchBox": {
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```
Comment (keller-mark) [inline src/viewconfigs/horizontal-multivec-6.json]:
```suggestion
          "chromInfoId": "hg38"
```

File: src/viewconfigs/horizontal-multivec-7.json [hunk 7]
```diff
@@ -15,6 +15,13 @@
           2671196920.8697634,
           2671320441.9350777
         ],
+        "genomePositionSearchBoxVisible": true,
+        "genomePositionSearchBox": {
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```
Comment (keller-mark) [inline src/viewconfigs/horizontal-multivec-7.json]:
```suggestion
          "chromInfoId": "hg38"
```

File: src/viewconfigs/horizontal-multivec-8.json [hunk 8]
```diff
@@ -15,6 +15,13 @@
             2671196920.8697634,
             2671320441.9350777
         ],
+        "genomePositionSearchBoxVisible": true,
+        "genomePositionSearchBox": {
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```
Comment (keller-mark) [inline src/viewconfigs/horizontal-multivec-8.json]:
```suggestion
          "chromInfoId": "hg38"
```

File: src/viewconfigs/horizontal-multivec-1.json (hunk 1 above)
Comment (sehilyi) [reply src/viewconfigs/horizontal-multivec-1.json]: Thanks for the suggestion.

**Pull request merged**